kubernetes/opentelemetry-demo.yaml

```diff
@@ -32,7 +32,7 @@ metadata:
   labels:
     app.kubernetes.io/name: otelcol
     app.kubernetes.io/instance: opentelemetry-demo
-    app.kubernetes.io/version: "0.82.0"
+    app.kubernetes.io/version: "0.86.0"
```

Comment:
This entire file is generated. Need to revert all changes on this file.

Reply:
What's the process for generating it?

Reply:
Reverted